chapters/3-measurement.tex

```diff
@@ -323,15 +275,17 @@ \subsection{Data flowcharts}
 the operations used to combine datasets did not create unwanted duplicates
 or incorrectly drop any observations.
 
+The information that goes into the data flowcharts can be expressed in text,
+but our experience is that diagrams are the most efficient way to communicate this information across a team.
 A data flowchart can be created in a flowchart drawing tool
 (there are many free alternatives online) or
-by using shapes in Microsoft PowerPoint.
+by Microsoft shapes or smartart.
```

Comment:
```suggestion
by Microsoft PowerPoint shapes or smartart.
```
Some word is missing right? Or what is "Microsoft shapes"?

Reply:
there are shapes and smartart in word as well as powerpoint which is why i hadn't specified; doing this in a word doc seems equally fine as ppt. how about: "by using the shapes or SmartArt tools in Microsoft Office."

Reply:
```suggestion
by using the shapes or SmartArt tools in, for example, Microsoft Office.
```
I like this the best. I added "for example" as we are not specifically recommending it. There are alternatives to office and we are just suggesting the is a an example.